maps_dashboards/public/components/layer_control_panel/layer_control_panel.tsx

```diff
@@ -315,6 +329,19 @@ export const LayerControlPanel = memo(
       );
     }
 
+    const getLayerTooltipContent = (layer: MapLayerSpecification) => {
+      const tooltipContent = `Layer type: ${layer.type}, Zoom range: ${layer.zoomRange[0]} - ${layer.zoomRange[1]}`;
+      if (layer.type === DASHBOARDS_MAPS_LAYER_TYPE.DOCUMENTS) {
+        return (
+          tooltipContent +
+          `, Index pattern: ${layer.source.indexPatternRefName}` +
+          `, Geo field: ${layer.source.geoFieldName}`
```

Comment:
why do we need these aditional details? IMO, lets say, layer is not visible outside zoom levels x - y  as content and keep layer name as title.

Reply:
Updated:

<img width="568" alt="image" src="https://user-images.githubusercontent.com/90288540/211113302-0a093359-2650-427f-800a-e985166332a1.png">

<img width="501" alt="image" src="https://user-images.githubusercontent.com/90288540/211113190-9cdb1cde-c817-425e-8e7a-23bc4d993d86.png">